src/main/java/com/cvent/foodflick/repositories/DinnerPartyRepository.java

```diff
@@ -1,9 +1,13 @@
 package com.cvent.foodflick.repositories;
 
 import com.cvent.foodflick.models.DinnerParty;
+import com.cvent.foodflick.models.dto.GetDinnerPartyDTO;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface DinnerPartyRepository extends JpaRepository<DinnerParty, Long> {
+    List<DinnerParty> findDinnerPartiesByCreatedBy(String userId);
```

Comment:
probably will delete this, but keeping in case we decide to filter dinner parties by host